content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+How we make our maps:
+
+When a builder decides that they want to make a map, they first of all, head to our (whitelisted) build server found at <code>build.nucleoid.xyz</code>
+
+They choose a game and create a map using a handy command.
+
+<br>
+<img class="header_image" src="assets/images/blog/Kg34kl.jpg">
```

Comment:
I believe this should work through markdown: `![Image Description](assets/images/Kg34kl.jpg)`

Reply:
Markdown results in the image being too big and overflow off the page. Hence the `header_image` class

Reply:
This should be resolved in the CSS then- an image within a blog overflowing the page is never desirable :D

Reply:
This is a blog post PR, not a CSS bug fix spree lol. I'll make a seperate PR with the fixes later on.